test/app/model/Rock.spec.ts

```diff
@@ -1,17 +1,28 @@
+import { Rock } from '../../../src/app/model/Rock';
 import { Square, Row, Column } from '../../../src/app/model/Types';
 
 describe('Rock movement', () => {
+  it('Check possibles values, when Rock is on A1', () => {
+    let board = generateEmptyBoard();
+    let rock = new Rock('1', 'WHITE');
+    let rockPosition: Square = { column: Column['A'], row: 1, piece: rock };
+
+    let rockPossibleMoves = rock.possibleMoves(rockPosition, board);
```

Comment:
@nowakprojects I need help ^_^
Rozumiem co jest nie tak, ale nie wiem jak to poprawić tutaj - w testach. Wgl ten Column jako enum to trochę nie pasuje, ale też nie widzę innej opcji, skoro chcę po nim iterować w metodach w klasie Rock. 
Chyba trochę zabłądziłem haha... Będę wdzięczny za sprostowanie.

Reply:
Zdaje mi się że tak :) Skoro ważne jest dla Ciebie co stoi na danym polu to wtedy pozostawiasz wolność w tym jak ktoś to zaimplementuje pod spodem.